src/fr/2017-03/06/02.md

```diff
@@ -1,5 +1,6 @@
 ---
-title:  La loi et la foi (Galates. 3:15–18)
+title:  La loi et la foi 
+(Galates. 3:15–18)
```

Comment:
@clope031 What I actually meant, is to move outside of `meta` area in to the lesson. Check out the English version